packages/components/pie-button/test/accessibility/pie-button.spec.ts

```diff
@@ -3,7 +3,8 @@ import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-obj
 import { ButtonComponent } from '../helpers/page-object/pie-button.component.ts';
 import { variants } from '../../src/defs.ts';
 
-variants.forEach((variant) => {
+// primary is excluded due to a11y issues
+variants.filter((variant) => variant !== 'primary').forEach((variant) => {
```

Comment:
q - are we planning to raise this with designers?

Reply:
I... didn't notice this got added. Think Ben must have been trying to diagnose a test problem unrelated. This shouldn't stay I don't think